packages/ember-core/src/services/flash-messages.ts

```diff
@@ -0,0 +1,79 @@
+import { A } from '@ember/array';
+import MutableArray from '@ember/array/mutable';
+import { action } from '@ember/object';
+import Service from '@ember/service';
+import { isBlank } from '@ember/utils';
+import { tracked } from '@glimmer/tracking';
+
+type FlashOptions = {
+  message?: string;
+  type?: 'success' | 'info' | 'warning' | 'danger' | undefined;
```

Comment:
The trailing `?` states that the property can be omitted or `undefined`.
```suggestion
  type?: 'success' | 'info' | 'warning' | 'danger';
```